doc/man3/ASN1_TIME_set.pod

```diff
@@ -11,6 +12,7 @@ ASN1_TIME_print, ASN1_TIME_diff - ASN.1 Time functions
  ASN1_TIME *ASN1_TIME_adj(ASN1_TIME *s, time_t t,
                           int offset_day, long offset_sec);
  int ASN1_TIME_set_string(ASN1_TIME *s, const char *str);
+ int ASN1_TIME_set_string_X509(ASN1_TIME *s, const char *str);
```

Comment:
Please add a history section mentioning this was added for OpenSSL 1.1.1.  There are examples on how to do this in other .pod files